nodejs/eks/cluster.ts

```diff
@@ -483,9 +483,9 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
     }
 
     let kubernetesNetworkConfig: pulumi.Output<aws.types.input.eks.ClusterKubernetesNetworkConfig> | undefined;
-    if (args.kubernetesServiceIpAddressRange) {
-        kubernetesNetworkConfig = pulumi.output(args.kubernetesServiceIpAddressRange).apply(
-            serviceIpv4Cidr => ({ serviceIpv4Cidr }),
+    if (args.kubernetesServiceIpAddressRange || args.kubernetesServiceIpFamily ) {
```

Comment:
do these not need to be different applies @viveklak ? What happens if a user doesn't set the serviceIpAddressRange but does specify the serviceIpFamily (and vice versa), we will try and set the other value

Reply:
Yeah this seems buggy but I have lost a bunch of context here. I would imagine you would want to include the ipv6 cidr for ipv6 family. I have lost a bunch of context here. @roothorp mentioned he would take this over and get it in.